approval_polls/templates/approval_polls/create.html

```diff
@@ -20,6 +20,7 @@
       {% endfor %}
     </ul>
 
+    <input type='hidden' name='username' value='{{ user.username }}'>
```

Comment:
We can not do this on the client side because then the application becomes insecure. A user can modify this on the client side using JavaScript and create a poll using some other user's name or a non-existent name.

This has to be done on the server side using user information in `request.session`